README.md

```diff
@@ -8,6 +8,8 @@ Formatter plugin for go template files. The only peer dependency is [prettier](h
 npm install --save-dev prettier prettier-plugin-go-template
 ```
 
+or see [here](#vscode) if you want to use this with vscode
```

Comment:
```suggestion
or see [VSCode](#vscode) if you want to use this with vscode
```

It's best to have links describe what they link to own their own, esp. for screen readers